mlocal/frags/go_normal_opts.mk

```diff
@@ -1,3 +1 @@
-# This tells go's link command to add a GNU Build Id, needed for later
-#   symbol stripping for example as is done by rpmbuild.
-GO_LDFLAGS += -ldflags="-B 0x`head -c20 /dev/urandom|od -An -tx1|tr -d ' \n'`"
```

Comment:
should look like to achieve this : `GO_LDFLAGS += -ldflags="-buildid="`

Reply:
It did not help. https://github.com/apptainer/apptainer/actions/runs/7100998731/job/19328214255?pr=1630 still fails with ` *** ERROR: No build ID note found in /home/testuser/rpmbuild/BUILDROOT/apptainer-1.2.5+377.gccb3226-1.el7.x86_64/usr/libexec/apptainer/bin/gocryptfs`